hack/release.sh

```diff
@@ -28,6 +28,7 @@ COMPONENTS=(
   ["camel.yaml"]="camel/source/config"
   ["kafka-importer.yaml"]="kafka/source/config"
   ["kafka-channel.yaml"]="kafka/channel/config"
+  ["natss.yaml"]="natss/config"
```

Comment:
Should this be called `natss-channel.yaml`? I know we don't have a NATSS importer, but it might be less confusing to users if all channels have the `-channel` suffix.

Reply:
/hold